### ISSUE TITLE
Create chapter for logistic Regression

### ISSUE THREAD

**Issue by MoritzGuck** [MoritzGuck/Machine-Learning-Reference#15]
Include: 
- Formula
- Use
- That it is a calibrated probability function for training data.
- Why regularization (l1 or l2) and early stopping is important -> keeps weights of lin model away from +/- infinity, helps logits (output of logistic regression away from 0 or 1 (asymtotes), which would halt training). 
-
